packages/camera/camera_android_camerax/test/android_camera_camerax_test.dart

```diff
@@ -466,69 +534,13 @@ void main() {
-    final MockRecorder mockRecorder = MockRecorder();
 
     // Tell plugin to create mock/detached objects for testing createCamera
     // as needed.
-    camera.proxy = CameraXProxy(
-      getProcessCameraProvider: () =>
-          Future<ProcessCameraProvider>.value(mockProcessCameraProvider),
-      createCameraSelector: (int cameraSelectorLensDirection) {
-        switch (cameraSelectorLensDirection) {
-          case CameraSelector.lensFacingFront:
-            return mockFrontCameraSelector;
-          case CameraSelector.lensFacingBack:
-          default:
-            return mockBackCameraSelector;
-        }
-      },
-      createPreview:
-          (ResolutionSelector? resolutionSelector, int? targetRotation) =>
-              Preview.detached(
-                  initialTargetRotation: targetRotation,
-                  resolutionSelector: resolutionSelector),
-      createImageCapture:
-          (ResolutionSelector? resolutionSelector, int? targetRotation) =>
-              ImageCapture.detached(
-                  resolutionSelector: resolutionSelector,
-                  initialTargetRotation: targetRotation),
-      createRecorder: (_) => mockRecorder,
-      createVideoCapture: (_) => Future<VideoCapture>.value(mockVideoCapture),
-      createImageAnalysis:
-          (ResolutionSelector? resolutionSelector, int? targetRotation) =>
-              ImageAnalysis.detached(
-                  resolutionSelector: resolutionSelector,
-                  initialTargetRotation: targetRotation),
-      createResolutionStrategy: (
-          {bool highestAvailable = false, Size? boundSize, int? fallbackRule}) {
-        if (highestAvailable) {
-          return ResolutionStrategy.detachedHighestAvailableStrategy();
-        }
-
-        return ResolutionStrategy.detached(
-            boundSize: boundSize, fallbackRule: fallbackRule);
-      },
-      createResolutionSelector: (ResolutionStrategy resolutionStrategy) =>
-          ResolutionSelector.detached(resolutionStrategy: resolutionStrategy),
-      createFallbackStrategy: (
-              {required VideoQuality quality,
-              required VideoResolutionFallbackRule fallbackRule}) =>
-          MockFallbackStrategy(),
-      createQualitySelector: (
-              {required VideoQuality videoQuality,
-              required FallbackStrategy fallbackStrategy}) =>
-          MockQualitySelector(),
-      createCameraStateObserver: (_) => MockObserver(),
-      requestCameraPermissions: (_) => Future<void>.value(),
-      startListeningForDeviceOrientationChange: (_, __) {},
-      setPreviewSurfaceProvider: (_) => Future<int>.value(
-          3), // 3 is a random Flutter SurfaceTexture ID for testing},
-    );
+    camera.proxy =
+        getProxyForTestingResolutionPreset(mockProcessCameraProvider);
 
-    when(mockProcessCameraProvider.bindToLifecycle(mockBackCameraSelector, any))
+    when(mockProcessCameraProvider.bindToLifecycle(any, any))
```

Comment:
I removed similar testing of this particular call in a couple of tests in this PR and felt comfortable doing so because it is tested elsewhere.